package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@digipolis/auth",
-  "version": "1.4.1",
+  "version": "1.4.3",
```

Comment:
1.4.2 seems to be missing ?
Also I would do a minor or even major because there are some behavioural changes.

Reply:
1.4.2 does exists here: https://github.com/digipolisantwerp/auth_module_nodejs/tree/v1.4.2

Fair point. It should be backwards compatible and (should have) none breaking changes.
So a minor would ok?

Reply:
If it's backwards compatible a minor is correct.

Reply:
Fixed (and pushed)